recipe/0001-base-path.patch

```diff
@@ -0,0 +1,319 @@
+Add base path support
```

Comment:
Taken from https://github.com/coder/code-server/blob/main/patches/base-path.diff.

Reply:
Removed all patches that couldn't be applied.